love/lib/supabase/answers.ts

```diff
@@ -0,0 +1,15 @@
+import { Row as rowFor, run } from 'common/supabase/utils'
+import { db } from 'web/lib/supabase/db'
+
+export const deleteAnswer = async (
```

Comment:
Have you added this ability in the `love_answers.sql`? I believe you need:
` drop policy if exists "self delete" on love_answers;
create policy  "self delete" on love_answers for delete
    using (creator_id = firebase_uid());`